extensions/registry.py

```diff
@@ -1820,6 +1820,12 @@
         'supporters' : { 'MESA' },
         'url' : 'extensions/EXT/EXT_EGL_sync.txt',
     },
+    'GL_EXT_fragment_shading_rate' : {
+        'esnumber' : 339,
```

Comment:
TODO: Update this based on the outcome of https://github.com/KhronosGroup/OpenGL-Registry/pull/509